api/Window.json

```
@@ -2402,24 +2405,26 @@
       "gamepaddisconnected_event": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/gamepaddisconnected_event",
-          "spec_url": "https://w3c.github.io/gamepad/#the-gamepaddisconnected-event",
+          "spec_url": [
+            "https://w3c.github.io/gamepad/#the-gamepaddisconnected-event",
+            "https://w3c.github.io/gamepad/#event-gamepaddisconnected"
+          ],
           "description": "<code>gamepaddisconnected</code> event",
           "support": {
-            "chrome": [
-              {
-                "version_added": "35"
-              },
-              {
-                "version_added": "21",
-                "version_removed": "35",
-                "prefix": "webkit"
-              }
-            ],
+            "chrome": {
+              "version_added": "35",
+              "partial_implementation": true,
+              "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
```

Here and below this change is needed:

```suggestion
              "notes": "The event handler property (<code>ongamepaddisconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
```